packages/matchbox/src/components/Toggle/tests/Toggle.test.js

```diff
@@ -18,7 +18,7 @@ describe('Toggle', () => {
   });
 
   it('renders default checked styles', () => {
-    wrapper = subject({ checked: true });
+    wrapper = subject({ checked: true, onChange: jest.fn() });
```

Comment:
Is this needed?

Reply:
Yea, onChange is required when a controlled value is set so it throws an error if it's not there